docs: Improve the local dev setup section. (#1575)
dynatrace-oss/barista: DEVELOPMENT.md

# Development

Thank you very much for trying to improve our design system! To get you up on

> lightning speed with our contribution here are some requirements that are needed

Reviewer: ```suggestion
lightning speed with your contribution, here are some requirements that are needed
```

for getting started.

## TLDR;

1. install [Node.js](https://nodejs.org/en/)
2. install [Bazel](https://docs.bazel.build/versions/3.5.0/install.html)
3. start coding 🚀

## Create a local development setup

### Install Node.js

To start contributing you need [Node.js](https://nodejs.org/en/) installed on

> your machine for macOS users we recommend doing that with

Reviewer: ```suggestion
your machine. For macOS users, we recommend doing that with
```

[nvm](https://github.com/nvm-sh/nvm) a node version manager. While Windows users
should go with the LTS version of Node.js.

> With Node.js **npm** the node package manager is shipped together.

Reviewer: ```suggestion
**npm** (Node Package Manager) is shipped with Node.js.
```

### Install Bazel

[Bazel](https://www.bazel.build/) is a build tool for incremental and hermetic
builds. We use it to build our Monorepository. Currently, we are in a transition

> from the AngularCli to Bazel so it is not fully migrated until now.

Reviewer: ```suggestion
from the AngularCli to Bazel so it is currently not fully migrated.
```
